todo_app/templates/todo_item.html

```diff
@@ -0,0 +1,21 @@
+{% extends "layout.html" %}
+{% block title %}To-Do App{% endblock %}
+{% block content %}
+  <div class="jumbotron">
+    <h1 class="display-4">To-Do App</h1>
+    <p class="lead">Just another to-do app.</p>
+  </div>
+  <div class="row">
+    <div class="col-md-12"></div>
+  </div>
+  <h2>Add Todo Items</h2>
+  <form action="" method="POST" class="add-todo">
+    <div class="add-todo">
+      <label for="todo_title">Enter Status: </label>
```

Comment:
Worth noting that your to-dos have a status field (e.g. "Not started"/"Completed") but that the field you actually save here is the name or title rather than the status